app/src/main/res/values-ar/strings.xml

```diff
@@ -365,6 +365,7 @@
   <string name="just_now">الآن</string>
   <string name="time_ago">منذ %s</string>
   <string name="yesterday">أمس</string>
+  <string name="recently">مؤخرا</string>
```

Comment:
Prefer not to add this or the Brazilian Portuguese variant--we'll have translators translate these after the English base string is checked in.

Reply:
Removed.